webpack.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    entry: [ "./public/scripts/main.js","./public/scripts/keymaster.js" ],
+    target: 'browser',
```

Comment:
I had to change target to `web` for it to load. If you run `webpack --version` what is shown? Curious if we might need to use `npx webpack` in package.json.